ansible.cfg

```diff
@@ -1,10 +1,12 @@
 [defaults]
 roles_path = ./community:./roles
 inventory = ./inventory/hosts
+forks = 10
```

Comment:
I read on the webpage you shared
"If you are orchestrating your playbook across many servers, then you may have noticed that the number of servers where the playbook executes simultaneously is 5."
but this is not true for us right? I think the deploy command runs on all our 8 servers at once, right?

Reply:
I'm not 100% sure of the details. I think each "task" Ansible executes actually involves several smaller operations behind the scenes. I assume they're not performed in a totally linear way when interacting with multiple servers. :man_shrugging: 

I'm provisioning `all-prod` now with forks set to 10, and I can see a lot of Ansible activity in `htop`. As far as I can tell there's lot of little processes reading from the playbooks, doing thing with ssh, and interacting with python modules locally. 

I think the 10 forks will help :+1:

Reply:
I see, cool!